riscv/mmu.cc

```diff
@@ -149,6 +149,23 @@ void mmu_t::load_slow_path(reg_t addr, reg_t len, uint8_t* bytes, uint32_t xlate
       throw *matched_trigger;
   }
 
+  if (unlikely(addr & (len-1))) {
```

Comment:
It seems this will never match as of 894ab32c66c4046d7ac405944e3a7b1ad4651fcc? That's not good because then `git bisect` wouldn't be able to identify this new code if it had a bug.

Reply:
I'm still trying to learn exactly what you're looking for in a set of changes. IIRC it was that each stage compiles, runs, and also is tiny. Can you point me at a list of all the rules somewhere?

I've squashed this change into another one.